test/test-bind-all.js

```diff
@@ -0,0 +1,146 @@
+var Lab = require('lab');
+var lab = exports.lab = Lab.script();
+
+var describe = lab.describe;
+var it = lab.it;
+var Code = require('code');
+var expect = Code.expect;
+
+var bindAll = require('../bind-all');
+
+describe('bind-all', function() {
+  it('should bind all the methods', function(done) {
+    var res;
+    var Test = function() {
+      this.run(this.fn);
+      bindAll(this);
+    };
+    Test.prototype = {
+      run: function(fn) { return fn(); },
+      fn: function() { res = this instanceof Test; }
+    };
+    new Test();
+
+    expect(res).to.be.true;
```

Comment:
`expect(res).to.be.true;` - `true` needs to be invoked for the assertion to actually be made. -> `expect(res).to.be.true();`. It's a `code` thing - I'm working on a PR to actually make `lab` check for these (should be soon)